src/redux/reducers/actionLiterals.ts

```diff
@@ -0,0 +1,9 @@
+export const actionLiterals = {
```

Comment:
Why one actionLiterals object instead of single object for each action?

Reply:
Can do that